api-2.0/src/test/java/org/openmrs/module/attachments/AttachmentsServiceTest.java

```diff
@@ -238,4 +245,66 @@ public void getAttachments_shouldReturnIsolatedAttachments() throws Exception {
 		    expectedAttachments.stream().map(Attachment::getUuid).collect(Collectors.toList()).toArray(),
 		    actualAttachments.stream().map(Attachment::getUuid).collect(Collectors.toList()).toArray());
 	}
+	
+	@Test
+	public void testIncludeIsolatedParameter() throws Exception {
```

Comment:
Respect the test routines naming convention.
→ `getAttachments_shouldBeConsistentAcrossServiceApi`